packages/react/src/types.ts

```diff
@@ -1,4 +1,4 @@
-import { ReactElement, ReactFragment, ReactPortal } from 'react';
+import type { ReactElement, ReactFragment, ReactPortal } from 'react';
```

Comment:
Should we remove the imported `ReactFragment` here too?

Reply:
Whoops, I was testing this by commenting it out and the tests passed and I forgot to check 🤣